.changeset/giant-dogs-crash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vue-storefront/next": major
+"@vue-storefront/nuxt": major
```

Comment:
It might be a `minor` for nuxt as there're no breaking changes.

Reply:
We could go with major, and use it to remove deprecated stuff from this module as the support for VSF prefix in runtime config.